c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -1554,14 +1906,14 @@ class DeviceCachingAllocator {
       segment_info.address = reinterpret_cast<int64_t>(head_block->ptr);
       segment_info.stream = head_block->stream;
       segment_info.is_large = (!head_block->pool->is_small);
-
+      segment_info.is_expandable = head_block->expandable_segment_;
```

Comment:
nit: `is_` I would think this is a bool, but it's a ptr

Reply:
is_expandable is a bool, this is like `if(head_block->expandable_segment_)`